### ISSUE TITLE
Usersnap Feedback - Organizers field not working - I can't enter more than 1 Primary Organizer[...]

### ISSUE THREAD

**Issue by ascott** [participedia/usersnaps#763]
**Sender**: scottowenfletcher@gmail.com
**Comment**: Organizers field not working - I can't enter more than 1 Primary Organizer/Manager. Each new selection just replaces the old one
[Open #462 in Usersnap Dashboard](https://usersnap.com/a/#/participedia-dev/p/participedia-201-a37e4cf6/462)

<a href='https://usersnappublic.s3.amazonaws.com/2019-06-27/23-13/363b7d0e-8a0b-47a1-a69f-0652b588023a.png'>![Usersnap Feedback - Organizers field not working - I can't enter more than 1 Primary Organizer(...)](https://usersnappublic.s3.amazonaws.com/2019-06-27/23-13/95e24914-d78e-429c-8c7d-e4f3e00245a3.jpg)</a>
<a href='https://usersnappublic.s3.amazonaws.com/2019-06-27/23-13/363b7d0e-8a0b-47a1-a69f-0652b588023a.png'>Download original image</a>

**Browser**: Chrome 75 (Windows 10)
**Referer**: [https://participedia.net/case/5861/edit?full=1](https://participedia.net/case/5861/edit?full=1)
**Screen size**: 1280 x 720 **Browser size**: 1920 x 867

Powered by [usersnap.com](https://usersnap.com/?utm_source=github_entry&utm_medium=web&utm_campaign=product)

**Comment by jesicarson** [comment]
@scottofletcher wouldn't the word "Primary" mean that this field only accepts one answer?

**Comment by ascott** [comment]
the autocomplete field component is designed to allow only a single selection.

**Comment by scottofletcher** [comment]
No, this is supposed to be a 'select all that apply' field. we can change it to 'organizers'. @plscully do you agree? many of our partners use this field to add their own organizations and I think it incentivizes collaboration on entries with more than one organizer. also, most cases have several primary organizers (often it's a gov agency/body that hires a specialist/practitioner to facilitate deliberations etc.)

**Comment by jesicarson** [comment]
Makes sense that multiple organizations could be involved. According to the explanatory text instructions in the case fields spreadsheet, users are supposed to use the field to enter one 'primary' organizer, then enter additional organizers in the narrative.

**Comment by scottofletcher** [comment]
oh did we say that? I think it makes more sense to have this as an open field. it will make the data more robust as well. I think what we meant is that 'supporters' can be left to the narrative

**Comment by plscully** [comment]
My recollection is in line with what Jesi noted. This was one of several places where we made a trade-off in favor of simplifying the data collection. If we allowed multiple organizers in the data, then it would be hard to tell who the primary organizer was. And it didn't make sense to rate them 1,2,3 b/c many projects don't set that sort of ranking. It's a collaborative effort. I think we should leave it as is. Users can always enter additional organization entries and link to them from the narrative.
